x-pack/plugins/cloud_security_posture/server/plugin.ts

```diff
@@ -67,20 +67,7 @@ export class CspPlugin
    * either directly after installation, or
    * when the plugin is started and a package is present.
    */
-  readonly #initializerTracker = (() => {
-    let initialized = false;
-    return {
-      get: () => initialized,
-      on: () => {
-        initialized = true;
-        this.logger.debug('initialized');
-      },
-      off: () => {
-        initialized = false;
-        this.logger.debug('not initialized');
-      },
-    };
-  })();
+  #isInitialized: boolean = false;
```

Comment:
it's a [private field](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes/Private_class_fields), makes it so that instances of `CspPlugin` cannot access `isInitialized`. only accessible from within the class itself. 

it's unlikely that a developer will ever change any fields in `CspPlugin` plugin instance, but IMO a safe guard is a good practice.